languages/elixir/exercises/concept/agent/.docs/after.md

```diff
@@ -0,0 +1,9 @@
+The choice to use the `Agent` module, or to use multiple processes at all, depends on the relationship to be modelled with the data. _Agent processes_ are useful when a shared state must be used by multiple processes.
```

Comment:
I was wondering if maybe it would be nice to have a very basic example on how to use agents in Elixir?